independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/SuppressForbidden.java

```diff
@@ -6,10 +6,13 @@
 import java.lang.annotation.Target;
 
 /**
- * Used to suppress forbidden-apis errors inside the element it's added to.
+ * Used to suppress forbidden-apis errors inside for the element it's added to.
```

Comment:
c&p.

Reply:
This has clearly been a typo that we've fixed in this case. I don't like adding it back.